base/exports.jl

```diff
@@ -1233,6 +1233,7 @@ export
     @html_str,
     @doc,
     @doc_str,
+    docstrloc,
```

Comment:
I don't think this should be exported from Base, if this is something we expect to primarily be used by tools like Documenter, DocStringExtensions, or the docsystem itself in how it displays docstrings